src/Core/Components/Dialog/Services/DialogService.cs

```diff
@@ -25,10 +25,10 @@ public Task CloseAsync(DialogReference dialog)
         return CloseAsync(dialog, DialogResult.Ok<object?>(null));
     }
 
-    public async Task CloseAsync(DialogReference dialog, DialogResult result)
+    public Task CloseAsync(DialogReference dialog, DialogResult result)
```

Comment:
I believe we had it like this for a reason but can't remember exactly. @dvoituron you still know?
We need to be sure there anren't any adverse effect of changing this. Have you done any testing?

Reply:
Yes, it's better to use the `return Task.CompletedTask` when no asynchronous method is included in this method. 
More details [here](https://stackoverflow.com/questions/13243975/suppress-warning-cs1998-this-async-method-lacks-await).